lib/msal-common/test/client/AuthorizationCodeClient.spec.ts

```diff
@@ -1388,12 +1389,39 @@ describe("AuthorizationCodeClient unit tests", () => {
                 account: testAccount,
                 correlationId: RANDOM_TEST_GUID,
                 postLogoutRedirectUri: TEST_URIS.TEST_LOGOUT_URI,
-                idTokenHint: "id_token_hint"
+                idTokenHint: "id_token_hint",
+                state: "test_state"
```

Comment:
Can we also add extraQueryParameters here to test? e.g. `extraQueryParamters: {param: "value"}`